### ISSUE TITLE
[SCREENREADER]: GIBCT EYB - IE11 MUST announce the updated live region after users press the Update benefits button

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#11300]
## [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Internet Explorer 11 + the JAWS screen reader (2019) do not announce the updated benefits panel when users press Update benefits. This is a challenge because it's working correctly in Chrome + JAWS, but IE11 misses. Screen reader users might not understand anything changed without that live region announcement. Screen shot attached below.

## Related Issues
* https://github.com/department-of-veterans-affairs/va.gov-team/issues/10199

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] IE11 + JAWS reads out the live region when users press the  `Update benefits` button on all profile types (higher learning, OTJ, VET TEC)

## Environment

* Windows 10
* Internet Explorer 11
* JAWS 2019 screen reader
* https://staging.va.gov/gi-bill-comparison-tool/profile/31801227 (this is my go-to test profile)

## Steps to Re-create
1. Start JAWS screen reader
1. Enter https://staging.va.gov/gi-bill-comparison-tool/profile/31801227 in browser
1. Tab or arrow to the Estimate your benefits section
1. Change 2-3 inputs so the Update benefits button is enabled
1. Press the Update benefits button
1. Verify the expand/collapse is focused and read aloud
1. Verify live region changes in the Estimate Your Benefits gray panel is not read aloud

## Possible Fixes (optional)
During PR review, I noticed there was a `setTimeout` callback on the function that focused the expand/collapse after users pressed Update benefits. I'm not sure this is the cause, but might be a place to start exploring.

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs

![Screen Shot 2020-07-16 at 9 59 31 AM](https://user-images.githubusercontent.com/934879/87687394-4a39d480-c74b-11ea-9e34-58e330fefc46.png)

**Comment by ksantiagomorgan6** [comment]
Looks like the Jaws developers are running into a similar issue https://github.com/FreedomScientific/VFO-standards-support/issues/408

**Comment by 1Copenut** [comment]
@ksantiagomorgan6 Agreed. I was able to confirm this issue independently with JAWS 2020. I commented on the original Freedom Scientific issue ( https://github.com/FreedomScientific/VFO-standards-support/issues/38#issuecomment-685833623 ) and so we appear to be at a standstill for now. I'll log this as a known issue in the full audit when I submit to the VA 508 office, and that the issue is upstream with Freedom Scientific.

**Comment by 1Copenut** [comment]
@desireeturner I reviewed this with JAWS 2021 and IE11, with the same outcome. The upstream issue @ksantiagomorgan6 linked to at Freedom Scientific is still open, so for all intents and purposes, we can't move forward on this item.

Chrome and Edge Chromium both announce the updated text as expected with JAWS 2021. I'm going to move this and #16002 to closed with the label `wontfix`.
